jekyll/_posts/2021-01-19-community-survey-results-2020.md

```diff
@@ -0,0 +1,221 @@
+[Rosetta code examples](http://rosettacode.org/wiki/Category:Nim) are also a popular choice as a learning resource, followed by ["Nim Basics"](https://narimiran.github.io/nim-basics/), ["Nim Days"](https://xmonader.github.io/nimdays/) and ["Nim Notes"](https://scripter.co/notes/nim).
+
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/18.png" alt="What kind of learning materials does Nim need?" style="width:100%"/>
+
+If anyone would like to create some more learning materials for Nim (which is very high on the community priority list, see below under the "Nim in 2021 and beyond" section), the most wanted types of learning materials are code examples and written tutorials.
+
+
+
+## Nim community and contributions
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/19.png" alt="Have you made code contributions to the Nim project in the past?" style="width:100%"/>
+
+Compared to 2019, we see a 20% increase of people who have already contributed to Nim!
+And there are almost three times more people who haven't contributed to Nim so far but plan to do that in the future.
+We're looking forward to this :)
+
+The most frequent reasons for not contributing (yet) are: lack of time and lack of skill/experience.
+
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/19a.png" alt="Did you make any donations to Nim?" style="width:100%"/>
+
+In the past we haven't stressed enough the importance of the [community donations](https://nim-lang.org/donate.html), which is reflected in this graph: only 10% of our users have made a donation in the past.
+We are happy to see that 2/3 of our users plan to do it in the future, which will help us to have more manpower to fix the most important bugs more promptly and work on some exciting new features.
+
+
+
+
+
+## Nim tools
+
+We've asked you how satisfied you are with the Nim tooling:
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/29.png" alt="What is your experience with the following tools? [nimble]" style="width:100%"/>
+<img src="{{site.baseurl}}/assets/news/images/survey2020/30.png" alt="What is your experience with the following tools? [choosenim]" style="width:100%"/>
+
+Most of our users use Nimble package manager and they are very satisfied by it.
+A smaller amount of users picked Choosenim as their way of installing and updating Nim versions, but those who did are quite satisfied with what it has to offer.
+
+
+<img src="{{site.baseurl}}/assets/news/images/survey2020/34.png" alt="What is your experience with the following tools? [nim doc]" style="width:100%"/>
+<img src="{{site.baseurl}}/assets/news/images/survey2020/33.png" alt="What is your experience with the following tools? [nimpretty]" style="width:100%"/>
+<img src="{{site.baseurl}}/assets/news/images/survey2020/32.png" alt="What is your experience with the following tools? [nimsuggest]" style="width:100%"/>
+
+Nim doc is generally liked by those who use it, as is Nimpretty.
```

Comment:
Who are the users of both?

Reply:
Sorry, we don't have names, the survey was anonymous. :P